test/OpenTelemetry.Instrumentation.AspNetCore.Tests/MetricTests.cs

```diff
@@ -27,6 +27,9 @@
 #if NET6_0
 using TestApp.AspNetCore._6._0;
 #endif
+#if NET7_0
+using TestApp.AspNetCore._7._0;
```

Comment:
If the test apps would use an identical namespace, this wouldn't be needed.
This is outside of the scope of your PR, but something to consider as a future enhancement.